components/molecule/photoUploader/src/fileTools.js

```diff
@@ -110,6 +110,7 @@ export const prepareFiles = ({
   currentFiles,
   defaultFormatToBase64Options,
   errorCorruptedPhotoUploadedText,
+  errorSaveImageEndpoint,
```

Comment:
Maybe `errorSaveImageText` it is similar to other props like: `errorCorruptedPhotoUploadedText`